src/app/store/selector/auth.selector.ts

```diff
@@ -0,0 +1,33 @@
+import { createSelector } from '@ngrx/store';
+import { IAppState } from '../state/app.state';
+import { IAuthState } from '../state/auth.state';
+
+const authState = (state: IAppState) => state.auth;
+
+export const selectUser = createSelector(
```

Comment:
The new standard that I have seen in most projects is that in the reducer file you also have the selectors (no need to separate them)